frontend/ui/widget/configdialog.lua

```diff
@@ -1003,9 +1002,6 @@ function ConfigDialog:onConfigChoose(values, name, event, args, events, position
             args = args or {}
             self:onConfigEvent(event, args[position], when_applied_callback)
         end
-        if events then
-            self:onConfigEvents(events, position)
```

Comment:
Not removing `function ConfigDialog:onConfigEvents(option_events, arg_index)` which seems no longer call by anything ?